portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase

```diff
@@ -587,7 +588,18 @@ definition {
 
 		CrossClusterReplication.setupCrossClusterReplication();
 
-		SearchAdministration.openSearchAdmin();
+		User.logoutPG(
+			password = "test",
+			userEmailAddress = "test@liferay.com");
+
+		User.loginPG(
+			nodePort = 9080,
+			password = "test",
+			userEmailAddress = "test@liferay.com");
+
+		CrossClusterReplication.setupCrossClusterReplication(portalCluster = "true");
```

Comment:
CCR doesn't need to be setup twice. The configs for the 8080 node should carry over to 9080. That being said, `ccrLocalClusterConnectionConfigurations` should be configured with `localhost:8080,localcluster` and `localhost:9080,localcluster` when there is a portal cluster.